contrib/Docker_Grafana_Blueflood/docker-entrypoint.sh

```diff
@@ -37,6 +37,8 @@ blueflood:
   apikey: $RAX_APIKEY                     
   authentication_module: rax_auth
   authentication_class: BluefloodAuth
+  enable_submetrics: true
+  submetric_aliases: { "_sum": 'sum' }
```

Comment:
@wolfdancer  If you only enable "sum", then thats the only submetric possible.  If someone wants to do "average" which has been the default so far, I dont believe its possible unless its added here.